documentation/src/components/code-example.ts

```diff
@@ -129,7 +129,7 @@ export class CodeExample extends LitElement {
         return html`
             ${this.showDemo
                 ? html`
-                      <div id="demo">
+                      <div class="demo-example">
```

Comment:
It seemed weird to have this be an `id` since we often have multiple on the page. I don't think this is causing any issues, but we should obviously do a good pass on the docs before merging.

Reply:
yeah -- gonna take me a while to get used to that. Will this make adding hashtag links harder out of curiousity? I see the `docs-page` element takes over.

Reply:
I'm fairly certain that you can't use these sorts of links to point to content in a shadow root. This is one of the reasons that the `westbrook/11ty` branch moves our content into the light DOM.